File: challenge6.js

const voterTurnout = (voter_signatures, voter_ids) => {
  // Code here!
  let truth = [];
  
  if (voter_signatures.length !== voter_ids.length){
    return false
  }
  
for (ii = 0; ii <= voter_ids.length; ii++){
  if (voter_signatures[ii] === voter_ids[ii]){
    truth[ii] = true
  } else {
    truth[ii] = false
  }
}

console.log(truth)

if (truth.every(Boolean)){
  return 'All clear, we can count the votes!'
} else {
  return 'FRAUD!'
}

}

File: ruby/challenge6.rb
def voterTurnout(voter_signatures, voter_ids)
    if (voter_signatures.length != voter_ids.length)
        return false
    end

    truth = []
    for ii in 0..voter_ids.length do
        if voter_signatures[ii] == voter_ids[ii]
            truth[ii] = true
        else 
            truth[ii] = false
        end
    end    
    
    if truth.any?
        return 'All clear, we can count the votes!'
    else
        return 'FRAUD!'
    end
    
end

# voter_signatures = ['Bill Billiamson','Kate Etak','Brandon Brandonus','Simon Simonson','Jane Janesford']
# voter_ids = ['Bill Billiamson','Kate Etak','Brandon Brandonus','Simon Simonson','Jane Janesford']       
# voterTurnout(voter_signatures, voter_ids)
